scripts/deb-builder-helpers.sh

```diff
@@ -335,13 +335,14 @@ build_archive_deb () {
   cp ./default/src/app/extract_blocks/extract_blocks.exe "${BUILDDIR}/usr/local/bin/mina-extract-blocks"
   
   mkdir -p "${BUILDDIR}/etc/mina/archive"
-  cp ../scripts/archive/download-missing-blocks.sh "${BUILDDIR}/etc/mina/archive"
+  cp ../scripts/archive/missing-blocks-guardian.sh "${BUILDDIR}/etc/mina/archive"
   
   cp ./default/src/app/missing_blocks_auditor/missing_blocks_auditor.exe "${BUILDDIR}/usr/local/bin/mina-missing-blocks-auditor"
   cp ./default/src/app/replayer/replayer.exe "${BUILDDIR}/usr/local/bin/mina-replayer"
   
   cp ../src/app/archive/create_schema.sql "${BUILDDIR}/etc/mina/archive"
-  
+  cp ../src/app/archive/drop_tables.sql "${BUILDDIR}/etc/mina/archive"
```

Comment:
Why is this added and wasn't here before?

Reply:
I noticed that we are not shipping drop_tables script which reverts create_schema.sql . It is common approach to ship both